api/v1/submissions/SubmissionHandler.inc.php

```diff
@@ -0,0 +1,344 @@
+				        $entry['metadata'] = array(
+                                                'caption'               => $submissionFile->getCaption(),
+                                                'credit'                => $submissionFile->getCredit(),
+                                                'copyrightOwner'        => $submissionFile->getCopyrightOwner(),
+                                                'permissionTerms'       => $submissionFile->getPermissionTerms(),
+				        );
+				}
+				$data[] = $entry;
+			}
+		}
+		catch (App\Services\Exceptions\InvalidSubmissionException $e) {
+			return $response->withStatus(404)->withJsonError('api.submissions.404.resourceNotFound');
+		}
+
+		return $response->withJson($data, 200);
+	}
+
+	/**
+	 * Retrieve participant list by stage
+	 *
+	 * @param $slimRequest Request Slim request object
+	 * @param $response Response object
+	 * @param array $args arguments
+	 *
+	 * @return Response
+	 */
+	public function getParticipants($slimRequest, $response, $args) {
+		$request = $this->getRequest();
+		$context = $request->getContext();
+		$data = array();
+
+		$sContainer = ServicesContainer::instance();
+		$submissionService = $sContainer->get('submission');
+
+		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+		$stageId = $slimRequest->getQueryParam('stageId', WORKFLOW_STAGE_ID_SUBMISSION);
+		$data = $submissionService->getParticipantsByStage($context->getId(), $submission, $stageId);
+
+		return $response->withJson($data, 200);
+	}
+
+	/**
+	 * Retrieve galley list
+	 *
+	 * @param $slimRequest Request Slim request object
+	 * @param $response Response object
+	 * @param array $args arguments
+	 *
+	 * @return Response
+	 */
+	public function getGalleys($slimRequest, $response, $args) {
+		$request = $this->getRequest();
+		$context = $request->getContext();
+		$data = array();
+
+		$sContainer = ServicesContainer::instance();
+		$submissionService = $sContainer->get('submission');
+
+		try {
+			$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+			$data = $submissionService->getGalleys($context->getId(), $submission);
+		}
+		catch (App\Services\Exceptions\SubmissionStageNotValidException $e) {
+			return $response->withStatus(400)->withJsonError('api.submissions.400.stageNotValid');
+		}
+
+		return $response->withJson($data, 200);
+	}
+
+	/**
+	 * Get submission metadata
+	 * @param $slimRequest Request Slim request object
+	 * @param $response Response object
+	 * @param array $args arguments
+	 * @return Response
+	 */
+	protected function submissionMetadata($slimRequest, $response, $args) {
+		$request = $this->_request;
+		$submission = $this->getAuthorizedContextObject(ASSOC_TYPE_SUBMISSION);
+		assert($submission);
+
+		$queryParams = $slimRequest->getQueryParams();
+		$format = isset($queryParams['format'])?$queryParams['format']:'';
+		import('plugins.metadata.dc11.schema.Dc11Schema');
+		if ($format == 'dc11' || $format == '') {
```

Comment:
Rather than hard-coding these 2 cases, it might be better to rely on the filter framework and its metadata capabilities.